imqsauth/http.go

```diff
@@ -770,3 +837,65 @@ func httpHandlerGetGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRe
 		httpSendGroupsJson(w, groups)
 	}
 }
+
+func (central *ImqsCentral) handleWebSocket(ws *websocket.Conn) {
+	permOK := false
```

Comment:
This code block is unnecessarily complex. It can be rewritten so that permOK and closeConnection are just a single variable. That should bring it down to about 5 lines.